src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
-  external json: t => 'a => unit = "" [@@bs.send];
+  external sendString : t => string => unit = "send" [@@bs.send];
+  external sendObject : t => Js.t 'a => unit = "send" [@@bs.send];
+  external sendBuffer : t => Buffer.t => unit = "send" [@@bs.send];
+  external sendArray : t => array 'a => unit = "send" [@@bs.send];
+  external json : t => 'a => unit = "" [@@bs.send];
 };
 
-let module Express = {
+module Express = {
   type t;
-  type middlewareT = (Request.t => Response.t => Next.t => unit);
+  type middlewareT = Request.t => Response.t => Next.t => unit;
   external express : unit => t = "" [@@bs.module];
   external use : t => middlewareT => unit = "" [@@bs.send];
   external static : path::string => middlewareT = "" [@@bs.module "express"];
   external get : t => string => ('a => Response.t => unit) => unit = "" [@@bs.send];
-  external listen: t => int => unit = "" [@@bs.send];
+  external listen : t => int => unit = "" [@@bs.send];
```

Comment:
You might want to consider adding a `port` label here, just for clarity. I don't think that would even sacrifice backwards compatibility, it can still be called without the label.